openrc/usbmuxd.in

```diff
@@ -0,0 +1,14 @@
+#!/sbin/openrc-run
+# Copyright 2023-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 or version 3
+
+name="USB multiplexing daemon"
+description="Socket daemon for the usbmux protocol used by Apple devices"
+command="@sbindir@/usbmuxd"
+command_args="--user usbmux"
```

Comment:
Specify flag for running in foreground. This becomes relevant when using `supervisor=supervise-daemon` in conf.d.

```suggestion
command_args="--user usbmux"
command_args_foreground="-f"
```